style.css

```diff
@@ -688,3 +739,587 @@
+    background-repeat: no-repeat;
+    background-position: top right;
+    background-position-y: 8.32vh;
+    background-size: 40.66vw;
+    height: 1195px;
+  }
+
+  .about-heading {
+    width: 566px;
+    height: 78px;
+    margin-top: 130px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 4.5rem;
+    line-height: 5.125rem;
+    letter-spacing: 0.37px;
+    color: #f55800;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .about-text {
+    width: 566px;
+    height: 148px;
+    margin-top: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.25rem;
+    line-height: 2rem;
+    color: #f4f5f7;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .resume {
+    margin-top: 24px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 161px;
+    height: 48px;
+    background: #f55800;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    margin-left: 142px;
+    margin-right: 142px;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+    border: none;
+  }
+
+  .horizontal-line {
+    margin-top: 139px;
+    margin-left: 142px;
+    margin-right: 142px;
+    margin-bottom: 146.73px;
+  }
+
+  .skill-list {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+
+  .head-skill-list {
+    text-align: center;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2rem;
+    line-height: 2.25rem;
+    margin-bottom: 32px;
+    color: #fff;
+  }
+
+  .semi-list {
+    list-style: none;
+    text-decoration: none;
+    text-align: center;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.375rem;
+    color: #fff;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .semi-list > li {
+    margin-bottom: 32px;
+  }
+
+  .contact-message-and-form {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .contact-message {
+    width: 455px;
+    text-align: left;
+    margin-top: 130px;
+    margin-bottom: 228px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    gap: 24px;
+    color: #262626;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .contact-form-section {
+    margin-top: 130px;
+    width: 684px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    list-style: none;
+  }
+
+  #name {
+    margin: 15px 16px;
+    width: 330px;
```

Comment:
- [ ] I think it's better to use the same width of the `<textarea>`. Will be more suitable for the Figma template.